src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+html {
+  margin: 0;
+  font-family: Avenir, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.page {
+  padding: 20px 48px;
+}
+
+.search__form {
+  display: flex;
+  flex-direction: column;
+}
+
+.search__form-small {
+  margin-top: 20px;
+}
+
+.input {
+  display: c;
+  outline: 0;
+  padding: 0;
+
+  color: #3d4e61;
+
+  width: 100%;
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+
+  position: relative;
+}
+
+.input--big {
+  height: 70px;
+  background-size: 19px;
+  background-position: 26px;
+  padding-left: 62px;
+}
+
+.input--small {
+  height: 42px;
+  background-size: 11px;
+  background-position: 13px;
+  padding-left: 33px;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.input:focus {
+  box-shadow: 0 4px 4px 0 #00000040;
+  background: url(images/Search.svg) linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+
+  font-weight: 900;
```

Comment:
you have not connected the 900 font

Reply:
and some problem i think Lineat-gradient doesn't work

Reply:
no, looks like the gradient works properly